fixtures/fun-generation/picospot-20-led.json

```diff
@@ -81,25 +81,32 @@
           "type": "Open"
         },
         {
-          "type": "Gobo"
+          "type": "Gobo",
+          "name": "Gobo 1"
         },
         {
-          "type": "Gobo"
+          "type": "Gobo",
+          "name": "Gobo 2"
         },
         {
-          "type": "Gobo"
+          "type": "Gobo",
+          "name": "Gobo 3"
         },
         {
-          "type": "Gobo"
+          "type": "Gobo",
+          "name": "Gobo 4"
         },
         {
-          "type": "Gobo"
+          "type": "Gobo",
+          "name": "Gobo 5"
         },
         {
-          "type": "Gobo"
+          "type": "Gobo",
+          "name": "Gobo 6"
         },
         {
-          "type": "Gobo"
+          "type": "Gobo",
+          "name": "Gobo 7"
```

Comment:
These names are the same ones as the auto-generated ones, so I'd rather not add them, as they don't really provide much value.

Reply:
No problem, removed :)